### PR TITLE
Backend: Fixed SkyHanni not being a mod

### PR DESCRIPTION
**Pull request by hannibal002**
## What
We were a tweaker only for couple commits. Now we are a tweaker and a mod. 🥳 

## Changelog Technical Details
+ Made SkyHanni into a Mod. - hannibal2 + nea

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/tweaker/ModLoadingTweaker.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        URL location = ModLoadingTweaker.class.getProtectionDomain().getCodeSource().getLocation();
+        if (location == null) return;
+        if (!"file".equals(location.getProtocol())) return;
+        try {
+            MixinBootstrap.getPlatform().addContainer(location.toURI());
+            String file = new File(location.toURI()).getName();
+            CoreModManager.getIgnoredMods().remove(file);
+            CoreModManager.getReparseableCoremods().add(file);
+        } catch (URISyntaxException e) {
+            System.err.println("NEU could not re-add itself as mod.");
```
Comment (jani270) [inline src/main/java/at/hannibal2/skyhanni/tweaker/ModLoadingTweaker.java]:
```suggestion
            System.err.println("SkyHanni could not re-add itself as mod.");
```

Comment (jani270) [reply src/main/java/at/hannibal2/skyhanni/tweaker/ModLoadingTweaker.java]: Could not guess that it was taken from neu

Comment (lineargraph) [inline src/main/java/at/hannibal2/skyhanni/tweaker/ModLoadingTweaker.java]: not neu

**Review by lineargraph — CHANGES_REQUESTED** [review]

File: src/main/java/at/hannibal2/skyhanni/tweaker/ModLoadingTweaker.java (hunk 1 above)
Comment (ItsEmpa) [reply src/main/java/at/hannibal2/skyhanni/tweaker/ModLoadingTweaker.java]: gg skyhanni is officially neu

**Pull request merged**